pkg/controllers/routingpolicy/routingpolicy_test.go

```diff
@@ -0,0 +1,181 @@
+
+	action := "list"
+	util.UpdateFakeClientCall(client, action, "ingressclasses", ingressClassList)
+	util.UpdateFakeClientCall(client, action, "ingresses", ingressList)
+	util.UpdateFakeClientCall(client, action, "services", testService)
+
+	informerFactory := informers.NewSharedInformerFactory(client, 0)
+	ingressClassInformer := informerFactory.Networking().V1().IngressClasses()
+	ingressClassInformer.Lister()
+
+	ingressInformer := informerFactory.Networking().V1().Ingresses()
+	ingressInformer.Lister()
+
+	serviceInformer := informerFactory.Core().V1().Services()
+	serviceLister := serviceInformer.Lister()
+
+	informerFactory.Start(ctx.Done())
+	cache.WaitForCacheSync(ctx.Done(), ingressClassInformer.Informer().HasSynced)
+	cache.WaitForCacheSync(ctx.Done(), ingressInformer.Informer().HasSynced)
+	cache.WaitForCacheSync(ctx.Done(), serviceInformer.Informer().HasSynced)
+	return ingressClassInformer, ingressInformer, serviceLister, client
+}
+
+func getLoadBalancerClient() client.LoadBalancerInterface {
+	return &MockLoadBalancerClient{}
+}
+
+type MockLoadBalancerClient struct {
+}
+
+func (m MockLoadBalancerClient) GetLoadBalancer(ctx context.Context, request ociloadbalancer.GetLoadBalancerRequest) (ociloadbalancer.GetLoadBalancerResponse, error) {
+	res := util.SampleLoadBalancerResponse()
+	return res, nil
+}
+
+func (m MockLoadBalancerClient) CreateLoadBalancer(ctx context.Context, request ociloadbalancer.CreateLoadBalancerRequest) (ociloadbalancer.CreateLoadBalancerResponse, error) {
+	//TODO implement me
+	panic("implement me")
+}
+
+func (m MockLoadBalancerClient) DeleteLoadBalancer(ctx context.Context, request ociloadbalancer.DeleteLoadBalancerRequest) (ociloadbalancer.DeleteLoadBalancerResponse, error) {
+	//TODO implement me
+	panic("implement me")
+}
+
+func (m MockLoadBalancerClient) GetWorkRequest(ctx context.Context, request ociloadbalancer.GetWorkRequestRequest) (ociloadbalancer.GetWorkRequestResponse, error) {
+	id := "id"
+	requestId := "opcrequestid"
+	return ociloadbalancer.GetWorkRequestResponse{
+		RawResponse: nil,
+		WorkRequest: ociloadbalancer.WorkRequest{
+			Id:             &id,
+			LoadBalancerId: &id,
+			Type:           nil,
+			LifecycleState: ociloadbalancer.WorkRequestLifecycleStateSucceeded,
+		},
+		OpcRequestId: &requestId,
+	}, nil
+}
+
+func (m MockLoadBalancerClient) CreateBackendSet(ctx context.Context, request ociloadbalancer.CreateBackendSetRequest) (ociloadbalancer.CreateBackendSetResponse, error) {
+	//TODO implement me
+	panic("implement me")
+}
+
+func (m MockLoadBalancerClient) UpdateBackendSet(ctx context.Context, request ociloadbalancer.UpdateBackendSetRequest) (ociloadbalancer.UpdateBackendSetResponse, error) {
+	//TODO implement me
+	panic("implement me")
+}
+
+func (m MockLoadBalancerClient) DeleteBackendSet(ctx context.Context, request ociloadbalancer.DeleteBackendSetRequest) (ociloadbalancer.DeleteBackendSetResponse, error) {
+	//TODO implement me
+	panic("implement me")
```

Comment:
We can leave this empty and remove panic.

Reply:
Ack